apps/docs/src/app/containers/component-docs/fabric/fabric.component.html

```diff
@@ -22,7 +22,36 @@ <h2>Dialog</h2>
 >
   {{ sampleContent2 }} {{ sampleContent3 }}
   <fab-dialog-footer>
-    <fab-default-button (onClick)="clickSave()" text="Save" contentStyle="margin-right: 10px;"></fab-default-button>
```

Comment:
Any reason this was removed?

Reply:
Yes. I was troubleshooting the issue that resulted in the change in the wrapper from changeDetectorRef.detectChanges() to markForCheck(). I'll comment more there. Reverted this.